integrationtest/neo/client/request/internal/GetAll.d

```diff
@@ -293,6 +293,7 @@ private scope class GetAllImpl
         Reader reader;
         Controller controller;
 
+        request_event_dispatcher.initialise(() { auto slices = new void[][1]; return &slices[0]; });
```

Comment:
Wrap that line.

Reply:
Also maybe add a comment to explain what this is about. "As the test client doesn't have a proper shared resources system, we just pass a new buffer."?

Ditto for other locations where this happens.